docs/index.md

```diff
@@ -66,6 +64,18 @@ $ mkdocs --version
 mkdocs, version 0.15.3
 ```
 
+#### Need to install Python or Pip?
+
+Install [Python] by downloading the installer from [https://www.python.org/downloads/](https://www.python.org/downloads/).
+
+> [pip] is already installed if you're using Python 2 >=2.7.9 or Python 3 >=3.4 downloaded from python.org, but you'll need to upgrade pip.
+
+If you using an older version of [Python] that you can't update and need to install [pip], recommend using the `get-pip.py` script. 
```

Comment:
This is not a complete sentence. Let's try:

> If you using an older version of [Python] and need to install [pip], you can download and run the `get-pip.py` script.

Then let's point to the script and tell people how to download it.